src/release/issue-triaging.md

```diff
@@ -75,25 +75,48 @@ Triaging them the same way as `needs-triage` is also useful.
+- [`C-*`]: Specifies the **category** of the label, for example a bug, tracking issue or discussion
+    - `A-diagnostics` issues usually don't have any `C-*` label.
     - Also note `C-optimization` for missed compiler optimizations.
-- `O-*`: For platform-specific issues, specifies the platform (architecture or operating system). For example macos, aarch64, windows
-- `A-*`: The areas that the issue is relevant to, for example linkage, patterns, diagnostics
-- `F-*`: When the issue concerns a specific (usually unstable) feature
+- [`O-*`]: For platform-specific issues, specifies the **platform** (architecture or operating system). For example `O-macos`, `O-aarch64`, `O-windows`.
+- [`A-*`]: The **areas** that the issue is relevant to, for example `A-linkage`, `A-patterns`, `A-diagnostics`.
+- [`L-*`]: When the issue concerns a specific **lint**.
+- [`F-*`]: When the issue concerns a specific (usually unstable, usually language) **feature**.
+- [`-Z*`]: When the issue concerns a specific unstable `-Z` compiler flag.
 - `requires-nightly`: This issue is not relevant to the stable compiler
 - `requires-{incomplete,internal}-features`: This issue requires an incomplete or internal feature. The latter often means that the issue
     should be closed in accordance with compiler [MCP 620](https://github.com/rust-lang/compiler-team/issues/620).
-- `regression-*`: Labels for tracking issues that are regressions.
-- `D-*`: Labels for diagnostics issue.
-- `I-*`: Different labels about the nature (originally, importance) of a bug. For example ICE, slow code, heavy code (binary size), crashes, unsoundness.
-  There are also some other `I-*` labels that don't really fit into this. For triaging, focus on `I-slow`, `I-heavy`, `I-ICE`, `I-crash`, `I-unsound`.
-- `P-*`: Priority labels. Applied using the [Compiler Prioritization procedure](../compiler/prioritization.md)
-- `S-*`: The status of an issue, for example S-needs-repro.
-- `E-*`: Calls for participation, for example to minimize an issue 
+- [`regression-*`]: Labels for tracking issues that are regressions.
+- [`D-*`]: Labels for **diagnostic** issues.
+- [`I-*`]: Different labels about the nature[^1] of a bug. For example ICE, slow code, heavy code (binary size), crashes, unsoundness.
+  There are also some other `I-*` labels that don't really fit into this. For triaging, focus on `I-ICE`, `I-crash`, `I-hang`, `I-slow`, `I-heavy`, `I-compiletime` and `I-unsound`.
+- [`P-*`]: **Priority** labels. Applied using the [compiler prioritization procedure](../compiler/prioritization.md).
+- [`S-*`]: The **status** of an issue, for example `S-needs-repro`.
+- [`E-*`]: Calls for participation[^2], for example to minimize an issue.
     - `E-mentor`: A mentor is available to help with the issue, which makes for good first issues.
     - `E-needs-mcve`: This issue has a reproduction, but it is not minimal, it should be minimized.
```

Comment:
There's also `E-needs-bisection` which might be worth mentioning

Reply:
Added.